spec/features/visitors/sign_up_spec.rb

```diff
@@ -15,7 +15,7 @@
   #   Then I see a message that I need to confirm my email address
   scenario 'visitor can sign up with valid email address' do
     sign_up_with('test@example.com')
-    expect(page).to have_content(
+    expect(page.body).to include(
```

Comment:
because the translation string contains html, we need to compare to source html (vs. just the rendered text)